website/docs/guides/working-with-guides/cloud/1-setting-up-examples-on-cloud.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Setting up example projects in dbt Cloud"
+id: 1-setting-up-examples-on-cloud
+slug: working-with-guides/cloud/1-setting-up
+description: Learn how to set up Guides example projects in dbt Cloud.
+displayText: dbt Cloud setup
+hoverSnippet: Learn how to set up Guides example projects in dbt Cloud.
+---
+
+## Getting started
```

Comment:
sugg removing the h2 header (otherwise when you view the page, it'll have a title followed by a header) but adding an intro like:

In this guide, you'll learn how to get set up with the new example projects in dbt Cloud.

```suggestion
## Getting started
```